charts/project-origin-registry/values.yaml

```diff
@@ -10,6 +10,30 @@ image:
   # tag is the tag to pull defaults to the chart version
   tag:
 
+# transactionProcessor holds the configuration for the transaction processor
+transactionProcessor:
+  # replicas defines the number of transaction processor containers to run
+  replicas: 3
+  # threads defines the number of parallel threads to run per transaction processor
+  threads: 5
+  # defines the weight in the consistent hash ring used to determine which transaction processor to send a transaction to.
+  weight: 10
```

Comment:
I believe this answers my other question, about how they are calculated. Manually it seems. Should it occur, that the throughput exceeds what can be handled, by the replica, I assume another replica will be spun up? 

Have you made any thoughts about whether:

More threads, per replica vs. a new replica, for handling high throughput?

Reply:
Currently this is done manually, automatic throttling would be a separate issue to implement.
Pros and cons, the number of queues in the hash-ring is replicas * threads. 
Replicas ensure one can do a rolling update and split the workload or more physical machines in a cluster. Threads is cheaper since an extra contrainer is not spun up.